### DISCUSSION TITLE
when will the windows version be released?

### DISCUSSION THREAD

**Discussion by liangbright** [pyg-team/pyg-lib#190] [General]
Today I deleted the old PyG on my Windows PC and tried to install the new one, and I find out that pyg-lib windows wheels are not available...  "Installation from Source" does not work...

**Comment by rusty1s** [comment]
Should be hopefully ready for PyG 2.3 (~1 month), sorry for the delay on this one. If it is urgent, you can also try to install `pyg-lib` from source on your own.
